commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/CPSearchResultsPortlet.java

```diff
@@ -171,8 +173,22 @@ protected void paginate(
 		Optional<Integer> paginationDeltaOptional =
 			paginationDeltaParameterValueOptional.map(Integer::valueOf);
 
+		int configurationPaginationDelta =
+			cpSearchResultsPortletInstanceConfiguration.paginationDelta();
+
+		Optional<PortletPreferences> portletPreferencesOptional =
+			portletSharedSearchSettings.getPortletPreferences();
```

Comment:
@alecsloan @marco-leo note that this line is incompatible with 7.2 -- the compatible method is `portletSharedSearchSettings.getPortletPreferences71`, but we haven't merged that change yet... so not sure if we want to address it here or in the other pull. Just FYI.

Reply:
@ethanbustad let's update the other pull, as I do think we will nee to fork for 7.2